### ISSUE TITLE
New metric: Jaccard Index

### ISSUE THREAD

**Issue by WWXkenmo** [theislab/atlas-feature-selection-benchmark#43]
Thanks for taking the time to suggest a new metric!

## Description

> _Please briefly describe the suggested metric: What is it? What does it measure? Why would it be a good fit for the project?_

The Jaccard index, or Jaccard similarity coefficient, defined as the size of the intersection divided by the size of the union of two label sets.

## Links

> _Links to information about the metric_

- **Description:** https://scikit-learn.org/stable/modules/generated/sklearn.metrics.jaccard_score.html?highlight=metric
- **Code:** https://github.com/scikit-learn/scikit-learn
- **Package:** https://pypi.org/project/sklearn/

## Anything else?

> _Anything else you think is important about the metric_

- Need to decide what kind of class averaging to use
